### PR TITLE
nixos/tests: add simple dockerTools test

### PR DESCRIPTION
**Pull request by Profpatsch**
A simple test to check whether `dockerTools` generates valid images.

### CODE REVIEW

**Comment by Profpatsch** [comment]
@grahamcofborg test docker-tools

**Comment by adisbladis** [comment]
@GrahamcOfBorg test docker-tools

**Comment by Profpatsch** [comment]
Yay!

File: nixos/release.nix [hunk 1]
```diff
@@ -244,6 +244,7 @@ in rec {
   tests.containers-macvlans = callTest tests/containers-macvlans.nix {};
   tests.couchdb = callTest tests/couchdb.nix {};
   tests.docker = callTestOnTheseSystems ["x86_64-linux"] tests/docker.nix {};
+  tests.docker-tools = callTest tests/docker-tools.nix {};
```
Comment (adisbladis) [inline nixos/release.nix]: Should we perhaps do `tests.docker-tools = callTestOnTheseSystems ["x86_64-linux"] tests/docker-tools.nix {};` or `tests.docker-tools = callTestOnTheseSystems lib.platforms.linux tests/docker-tools.nix {};`

Comment (Profpatsch) [reply nixos/release.nix]: To save on testing resources?

**Comment by Profpatsch** [comment]
@GrahamcOfBorg test docker-tools

**Comment by Profpatsch** [comment]
Looks like ofborg doesn’t look at the `callForTheseSystems` flags when evaluating tests.

**Pull request merged**